core/commands/files.go

```diff
@@ -94,6 +94,7 @@ cache, free memory and speed up read operations.`,
 const (
 	filesCidVersionOptionName = "cid-version"
 	filesHashOptionName       = "hash"
+	filesNoValidateOptionName = "no-validate"
```

Comment:
nit: rename to match `--force` we have in other places.
```suggestion
	filesForceOptionName = "force
```

Reply:
@lidel I had changed this from `"force"` because CI complained about the same flag being used in multiple places.